package-lock.json

```diff
@@ -15204,213 +15408,168 @@
       }
     },
     "node_modules/verdaccio": {
-      "version": "5.21.1",
-      "resolved": "https://registry.npmjs.org/verdaccio/-/verdaccio-5.21.1.tgz",
-      "integrity": "sha512-SbqeKxmcUW1G9AYo8cmAPAlWW6YpNR8Q6LVJrfP+4s9gQ496s8cvhj0yAX8nl1k3+DAEZBuGcrgfAF0kORYXpA==",
+      "version": "5.14.0",
+      "resolved": "https://registry.npmjs.org/verdaccio/-/verdaccio-5.14.0.tgz",
+      "integrity": "sha512-++YTBxeUvBcsZb3e77x2lH+bdg5xrETi7h+5xtd2KPHrcW+MlpwCWDcwyHdCVZ7LhOgkzSSJD9L/0i1BkbwB8Q==",
```

Comment:
This is the Key Change.
Verdaccio is pinned to 5.14.0.
It could be that we should change the line in the package.json
to be ` "verdaccio": "5.14.0",`.
@seebees would changing the package.json to remove `^` for verdaccio prevent future us from updating verdaccio with `npm audit` etc.?

(All of this commit's change was generated by `npm install --dev verdaccio@5.14.0`)